src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1488,9 +1488,9 @@ internal enum ErrorCode
         ERR_NoRefOutWhenRefOnly = 8308,
         ERR_NoNetModuleOutputWhenRefOutOrRefOnly = 8309,
         ERR_BadOpOnNullOrDefault = 8310,
-        ERR_BadDynamicMethodArgDefaultLiteral = 8311,
+        // available 8311,
```

Comment:
>// available [](start = 8, length = 12)

This doesn't look right. We do not reuse ids for obsolete errors/warnings that were shipped. See ERR_BadParameterModifiersOrder, for example.